doc/source/_toc.yml

```diff
@@ -99,32 +100,19 @@ parts:
       - file: ray-core/examples/overview
         title: "Tutorials and Examples"
 
-  - caption: Deploying Ray Clusters
+  - caption: Ray Clusters | Deployments
```

Comment:
FYI, the caption says "Clusters" here but elsewhere we have "Ray Cluster", should we make that consistent?